velox/docs/develop/scalar-functions.rst

```diff
@@ -582,6 +582,58 @@ are created and then when they are copied again to the Velox vector. Some work i
 avoid that by using writer proxies that write directly to Velox vectors. This section will
 be updated once the new writer interfaces are completed.
 
+Variadic Arguments
+^^^^^^^^^^^^^^^^^^
+
+The last argument to a simple function may be marked "Variadic". This means
+invocations of this function may include 0..N arguments of that type at the end
+of the call.  While not a true type in Velox, "Variadic" can be thought of as a
+syntactic type, and behaves somewhat similarly to Array.
+
+==========  ==============================  =============================
+Velox Type  C++ Argument Type               C++ Actual Argument Type
+==========  ==============================  =============================
+VARIADIC       arg_type<Variadic<E>>              VariadicView<VectorOptionalValueAccessor<VectorReader<E>>>>
+==========  ==============================  =============================
+
+Like the ArrayView, VariadicView has a similar interface to
+*const std::vector<std::optional<V>>*.
+
+VariadicView supports the following:
+
+- size_t size() : return the number of arguments that were passed as part of the "Variadic" type in the function invocation.
+
+- VectorOptionalValueAccessor<arg_type<T>> operator[](size_t index) : access the value of the argument at index.
+
+- VariadicView<T>::Iterator begin() : iterator to the first argument.
+
+- VariadicView<T>::Iterator end() : iterator indicating end of iteration.
+
+- bool mayHaveNulls() : a check on the nullity of the arugments (note this takes time proportional to the number of arguments). When it returns false, there are definitely no nulls, a true does not guarantee null existence.
```

Comment:
I realized while writing this that this function is pretty useless (I copied it from ArrayView where it has performance benefits).  I should probably just remove it.

Reply:
* I'm referring to mayHaveNulls()